image_writers_guide/guidelines.adoc

```diff
@@ -5,143 +5,186 @@
 :icons:
 :experimental:
 :toc: macro
-:toc-title: 
+:toc-title:
 
 toc::[]
 
 == Overview
-When creating Docker images to run on OpenShift there are a number of best practices to consider in order to ensure consumers of those images have a good experience.  Since images are intended to be immutable and used as is, it is the image author's responsibility to ensure the image contains any artifacts that would typically be needed by a user of the image and that the image can be configured by the user.  Here are some general comments around things to consider.
+When creating Docker images to run on OpenShift, there are a number of best practices to consider as an image author to ensure a good experience for consumers of those images. Because images are intended to be immutable and used as is, it is your responsibility to ensure:
 
-== General Docker Best Practices
-The following are some guidelines that apply to Docker image creation in general, independent of whether the images are used on OpenShift v3.
+- the image contains any artifacts that would typically be needed by a user of the image, and
+- the image can be configured by the user.
 
-=== Reuse images
-Whereever possible you should base your image on an appropriate upstream image via the `FROM` statement.  This will ensure that your image can easily pick up security fixes
-from an upstream image when it is updated, rather than you being responsible for updating your dependencies directly.  
+== General Docker Guidelines
+The following are some guidelines that apply to Docker image creation in general, independent of whether the images are used on OpenShift. See the official https://docs.docker.com/articles/dockerfile_best-practices/[Docker documentation] for more comprehensive guidelines.
```

Comment:
We can also include here Atomic's guidelines, since it contains a couple of tips worth knowing about: http://www.projectatomic.io/docs/docker-image-author-guidance/